testing/scenario_app/README.md

```diff
@@ -42,6 +42,18 @@ compared against golden reside.
 
 ## Running for Android
 
+The test is run on a x86 emulator. To run the test locally, you must create an emulator running API level 28, and set the following screen settings in the avd's `config.ini` file:
```

Comment:
I think there is an ongoing effort to move to api level 29 do we want to update this to 29?

Reply:
This sounds doable. I can coordinate that the Chrome infra folks.